### ISSUE TITLE
Table pragma has no effect on table name

### ISSUE THREAD

**Issue by dom96** [moigagoo/norm#69]
I think this may be a regression, sorry but I don't have time to reproduce a minimal example:

```nim
  type
    User* {.table: "users".} = object
      nickname*: Option[string] ## Set via `/user/@id/set_nickname`
      email*: string

    SkinsOwnership* {.table: "skins_ownership".} = object
      skin* {.dbType: "INTEGER", parseIt: it.i.Skin, formatIt: ?it.int.}: Skin
      acquireTime* {.
        dbType: "INTEGER",
        parseIt: it.i.fromUnix().utc(),
        formatIt: ?it.toTime().toUnix()
      .}: DateTime
      userId* {.
        fk: User,
        onDelete: "CASCADE"
      .}: int
```

This ends up with:

```
DEBUG CREATE TABLE user (
        id INTEGER NOT NULL DEFAULT 0 PRIMARY KEY,
        nickname TEXT,
        email TEXT NOT NULL DEFAULT '',
)
DEBUG CREATE TABLE skinsownership (
        id INTEGER NOT NULL DEFAULT 0 PRIMARY KEY,
        skin INTEGER,
        acquireTime INTEGER,
        userId INTEGER NOT NULL DEFAULT 0 REFERENCES users (id) ON DELETE CASCADE
)
```

Note the REFERENCES pointing to `users` whereas the table name is `user`.

This used to work, but recently started failing for me once I inserted data into `SkinsOwnership`. Workaround is to remove the `{.table: "users".}`

**Comment by moigagoo** [comment]
Note to self:
- https://github.com/moigagoo/norm/blob/develop/src/norm/sqlite/sqlgen.nim#L28

**Comment by moigagoo** [comment]
Fixed in 1.1.3.
